frontend/src/pages/NewStudent.vue

```diff
@@ -0,0 +1,306 @@
+<template>
+  <div class="container">
+    <div class="Title">
+        <b class="newstudent">New Student</b>
+      <!-- upload button --> 
+        <b-field class="file is-primary is-right" :class="{'has-name': !!file}">
+            <b-upload v-model="file" class="file-label">
+                <span class="file-cta">
+                    <b-icon class="file-icon" icon="upload"></b-icon>
+                    <span class="file-label">  Bulk upload CSV</span>
+                </span>
+                <span class="file-name" v-if="file">
+                    {{ file.name }}
+                </span>
+            </b-upload>
+        </b-field>   
+    </div>
+    <div class="columns is-multiline is-mobile">
+      <!--start of first column (image) --> 
+      <div class="column is-one-third" align="center">
+        <img src="../assets/student.png" />
+      </div>
+      <!-- Start of 2nd column (all input fields) --> 
+      <div class="column is-two-thirds">
+        <section>
+            <b-field label="Name" class="half-width">
+                <b-input v-model="name"></b-input>
```

Comment:
a `name` property is referenced here without being defined. this causes an error in our browser console when we serve the app (verify this):

```
[Vue warn]: Property or method "name" is not defined on the instance but referenced
during render. Make sure that this property is reactive, either in the data option,
or for class-based components, by initializing the property. 
See: https://vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.

found in

---> <NewStudent>
       <Sidebar>
         <App>
           <Root>
```

to fix this, add a `name` property to the data object.